airflow/providers/amazon/aws/transfers/s3_to_redshift.py

```diff
@@ -90,27 +93,39 @@ def __init__(
         self._s3_hook = None
         self._postgres_hook = None
 
+    def _build_copy_query(self, credentials, copy_options):
```

Comment:
Please add type hints

Reply:
One is for the `unload` query while the other one is for the `copy` query.

Reply:
Is there a common logic that can be worth extracting?

Reply:
I think so but where to put it? We don't have a `RedshiftHook` because most of it can be done via `PostgresHook`. In this case we could add `redshift.py` to `providers.amazon.aws.utils` ?

Reply:
Will do, thanks.

Reply:
Added in the last commit.